setup.py

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env python3
-
-# Start ignoring PyImportSortBear as imports below may yield syntax errors
-from bears import assert_supported_version
```

Comment:
I would do something like the following

    from subprocess import call
    import locale
    import sys

    from setuptools import find_packages, setup
    from setuptools.command.test import test as TestCommand
    import setuptools.command.build_py

    from bears import assert_supported_version
    from bears import Constants

As `subprocess, sys, locale` are `python's` own library they come first. then `setuptools`, which is third party library and that is nether `python's` nor local to our repo and then comes our local imports.

Also do not forget to check the comments appear in the right place